### PR TITLE
New authors: Dr. Kelvin Droegemeier

### PR DESCRIPTION
**Pull request by DanielJPino-ux**
adding author to index

A one-line description of the changes you're making and how they'll affect users.

---

**Preview:**

### CODE REVIEW

**Comment by DanielJPino-ux** [comment]
https://federalist-466b7d92-5da1-4208-974f-d61fd4348571.app.cloud.gov/preview/gsa/digitalgov.gov/Dr-Kelvin-Droegemeier---Author/authors/dr-kelvin-droegemeier/

File: content/authors/dr-kelvin-droegemeier/_index.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+bio: "Director of the White House Office of Science and Technology Policy"
+
+# bio_url — Where can people learn more about your work? Provide a full URL [e.g. 'https://www.example.gov/']
+bio_url: https://www.whitehouse.gov/people/kelvin-k-droegemeier/
+
+# Agency Full Name [e.g. U.S. General Services Administration]
+agency_full_name: "White House"
+
+# Agency Acronym [e.g., GSA]
+agency: "WHOSTP"
```
Comment (saracope) [inline content/authors/dr-kelvin-droegemeier/_index.md:27]: This should just be OSTP

**Comment by DanielJPino-ux** [comment]
@ToniBonittoGSA - can you review this one? I checked the files changed and the changes were made, but the build preview hasn't reflected the change.

**Comment by DanielJPino-ux** [comment]
that worked! https://federalist-466b7d92-5da1-4208-974f-d61fd4348571.app.cloud.gov/preview/gsa/digitalgov.gov/Dr-Kelvin-Droegemeier---Author/authors/dr-kelvin-droegemeier/

**Pull request merged**